src/inference/dev_api/openvino/runtime/isync_infer_request.hpp

```diff
@@ -150,17 +150,20 @@ class OPENVINO_RUNTIME_API ISyncInferRequest : public IInferRequest {
 
     std::unordered_map<std::shared_ptr<ov::descriptor::Tensor>, std::vector<ov::Tensor>> m_batched_tensors;
 
+    /**
+     * @brief Finds input or output port
+     * @return structure which contains index of Input/Output or report that port wasn't found
+     */
+    // TODO vurusovs: TEMPORARY for ov::hetero::InferRequest::InferRequest()
+    FoundPort find_port(const ov::Output<const ov::Node>& port) const;
```

Comment:
The initial idea was that user shouldn't use this API, get/set tensor should be enough.